Source/Components/StackCluster.swift

```diff
@@ -35,11 +35,14 @@ extension ClusterLayout {
             let title = meta["title"] as? String
             let spacing = (meta["spacing"] as? NSNumber)?.floatValue
             let preserveParentWidth = meta["preserve_parent_width"] as? Bool
-            let axisRawValue = meta["axis"] as? Int
-            let axis = axisRawValue.flatMap { UILayoutConstraintAxis(rawValue: $0) }
+            var axis = UILayoutConstraintAxis.vertical
+            if let orientationRawValue = meta["orientation"] as? String,
+                let orientation = Orientation(rawValue: orientationRawValue) {
+                axis = orientation.layoutConstraintAxis
+            }
```

Comment:
If you really think it's useful to keep it for the parsing ok, yes I would keep it private.